.gitignore

```diff
@@ -67,3 +67,6 @@ temp
 # Rush files
 common/temp/**
 package-deps.json
+
+# Keep temp folders in mocked 'repos' that are used for unit tests
+!**/test/**/temp
```

Comment:
Can this be moved to a `.gitignore` just for `rush-lib`?

Reply:
Yes.